src/components/form/form.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import Form from '.'
+import Input from '../input'
+import Text from '../text'
+
+describe('<Form.Login /> component', () => {
+  let root
+  const mockCallBack = jest.fn()
+  const error = 'error'
+
+  beforeAll(() => {
+    root = shallow(<Form.Login onSubmit={mockCallBack} error={error}/>)
+  })
+
+  test('should be defined', () => {
+    expect(root.exists()).toBeTruthy()
+  })
+  test('should have user input', () => {
+    expect(root.find(Input.Text)).toHaveLength(1)
+  })
+  test('should have password input', () => {
+    expect(root.find(Input.Password)).toHaveLength(1)
+  })
+  test('should have error text', () => {
+    expect(root.find(Text.Error)).toHaveLength(1)
+    expect(root.find(Text.Error).dive().text()).toMatch(error)
+  })
+  test('test submit event', () => {
```

Comment:
mensagem de test não precisa avisar que é um test, pois é uma informação contextual
sugestão: `should call prop when internal form submit`